erica/domain/Shared/BaseDomainModel.py

```diff
@@ -9,12 +9,12 @@
 
 class AuditedModel(BaseModel):
     created_at: datetime.datetime
-    updated_at: datetime.datetime
+    updated_at: Optional[datetime.datetime]
```

Comment:
Fine, but it doesn't need to be optional. When the entity is created, the updated value is the same as created and later only the updated field is changed.

Reply:
Yes. However, I think that it is better to set the updated_at via SQLAlchemy (basically always updating updated_at, when we update it. Therefore, we would not need to set it in the beginning.


But we could